frontend/components/reference_component/index.js

```diff
@@ -17,9 +16,9 @@ function ReferenceComponent(props) {
   // use effect (called when component loads)
   useEffect(() => {
     const getUser = async () => {
-      const user = await referenceController.getUser('5eb9f120b582611adc9914c0');
-      setUser(user);
-    }
+      // const user = await referenceController.getUser('5eb9f120b582611adc9914c0');
```

Comment:
Do you need this commented out code in the future? If not, you should just remove it

Reply:
Done